pkg/apis/config/types.go

```diff
@@ -78,6 +78,8 @@ type MetricProviderSpec struct {
 	Address string
 	// The authentication token of the metric provider
 	Token string
+	// Whether to enable the InsureSkipVerify options for https requests on Metric Providers.
+	InsecureSkipVerify bool
```

Comment:
Can you check the UT in apis/config/scheme/scheme_test.go to ensure:

- by passing in a non-default boolean value, it ends with the passed-in value
- by not providing this value, it ends up with a defaulted value

Reply:
@Huang-Wei , thanks for the reminder. I added the UT for the field in both encoding and decoding tests.

Reply:
And I also add the test in [cmd/scheduler/main_test.go](https://github.com/kubernetes-sigs/scheduler-plugins/pull/341/commits/4a78c35c49db51d64e45eea1930e5fbf39d954cd#diff-d122acc1cc4c576c8e5174000be98882b805b006fab403dfdeefe0886ba5385e)